pkg/accounting/rollup/rollup_test.go

```diff
@@ -8,79 +8,122 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
-	"go.uber.org/zap"
 
 	"storj.io/storj/internal/testcontext"
-	"storj.io/storj/internal/teststorj"
-	"storj.io/storj/pkg/accounting/rollup"
+	"storj.io/storj/internal/testplanet"
 	"storj.io/storj/pkg/storj"
-	"storj.io/storj/satellite"
-	"storj.io/storj/satellite/satellitedb"
 )
 
-func TestQueryOneDay(t *testing.T) {
-	// TODO: use testplanet
+type testFlags struct {
+	days   int
+	atRest float64
+	bw     []int64
+}
 
-	ctx, r, db, nodeData, cleanup := createRollup(t)
-	defer cleanup()
+func TestOneDay(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 10, UplinkCount: 0,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		flags := &testFlags{
+			days:   1,
+			atRest: float64(5000),
+			bw:     []int64{1000, 2000, 3000, 4000},
+		}
 
-	now := time.Now().UTC()
-	later := now.Add(time.Hour * 24)
+		time.Sleep(time.Second * 2)
 
-	err := db.Accounting().SaveAtRestRaw(ctx, now, nodeData)
-	assert.NoError(t, err)
+		saveAndQuery(t, ctx, flags, planet)
+	})
+}
 
-	// test should return error because we delete latest day's rollup
-	err = r.Query(ctx)
-	assert.NoError(t, err)
+func TestTwoDays(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 10, UplinkCount: 0,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
 
-	rows, err := db.Accounting().QueryPaymentInfo(ctx, now, later)
-	assert.Equal(t, 0, len(rows))
-	assert.NoError(t, err)
-}
+		flags := &testFlags{
+			days:   2,
+			atRest: float64(5000),
+			bw:     []int64{1000, 2000, 3000, 4000},
+		}
 
-func TestQueryTwoDays(t *testing.T) {
-	// TODO: use testplanet
+		time.Sleep(time.Second * 2)
 
-	ctx, _, db, nodeData, cleanup := createRollup(t)
-	defer cleanup()
+		saveAndQuery(t, ctx, flags, planet)
+	})
+}
 
-	now := time.Now().UTC()
-	then := now.Add(time.Hour * -24)
+func TestThreeDays(t *testing.T) {
```

Comment:
I think a single test across 3 days would be sufficient, not sure whether we gain something from having more. Alternatively it could be a single test and testplanet where in each loop iteration adds one additional day.